drivers/volume/portworx/portworx.go

```diff
@@ -71,6 +94,8 @@ const (
 	validateClusterStartTimeout      = 2 * time.Minute
 	validatePXStartTimeout           = 5 * time.Minute
 	validateNodeStopTimeout          = 5 * time.Minute
+	validateStoragePoolSizeTimeout   = 3 * time.Hour
```

Comment:
Create a PTX to make this a function of the workload size. A static 3 hour timeout is too large for small workloads.